routes/index.js

```diff
@@ -29,11 +30,29 @@ routes.POSTregister = function(req, res, next) {
     if (err) {
       console.log('error while user register!', err); return next(err);
     }
-    console.log('user registered!');
-
-    passport.authenticate('local')(req, res, function () {
-      res.send({redirect: '/'});
+    
+    //send email verification
+    //Used example: https://github.com/heitortsergent/passport-local-mongoose-email/tree/master/examples/login
+    var authenticationURL = 'http://localhost:3000/verify?authToken=' + user.authToken;
```

Comment:
Oh that's a great point! I totally missed this. This should probably be an
environment variable that's different on our machines vs heroku so it uses
the correct url.

On Wed, Mar 30, 2016 at 10:26 AM Annabel Consilvio <notifications@github.com>
wrote:

> In routes/index.js
> <https://github.com/davidabrahams/Spring-Initiative/pull/15#discussion_r57897304>
> :
>
> > @@ -29,11 +30,29 @@ routes.POSTregister = function(req, res, next) {
> >      if (err) {
> >        console.log('error while user register!', err); return next(err);
> >      }
> > -    console.log('user registered!');
> > -
> > -    passport.authenticate('local')(req, res, function () {
> > -      res.send({redirect: '/'});
> > +
> > +    //send email verification
> > +    //Used example: https://github.com/heitortsergent/passport-local-mongoose-email/tree/master/examples/login
> > +    var authenticationURL = 'http://localhost:3000/verify?authToken=' + user.authToken;
>
> Should this be local host or the heroku page? Or should it be able to
> switch between both? (I'm actually not sure which is why I'm asking)
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/davidabrahams/Spring-Initiative/pull/15/files/f73a98f02067434bbce64c607c70e1419d57a67e#r57897304>
>